app/src/components/TimestampOrderBookScroller.js

```diff
@@ -59,7 +60,19 @@ class TimestampOrderBookScroller extends Component {
             });
         });
 
-        this.setState({listItems}, () => {
+        if (listItems.length > 0){
+            listItems.map(listItem =>{
```

Comment:
We can probably do this in a single pass in the `map`s on lines 55 & 47

Reply:
sure